src/objects/Account.object

```diff
@@ -9,6 +9,20 @@
         <label>HEDA Account Compact Layout</label>
     </compactLayouts>
     <fields>
+        <fullName>Current_Address__c</fullName>
+        <deleteConstraint>SetNull</deleteConstraint>
+        <description>The current Address that the Contact&apos;s mailing fields are filled with.</description>
```

Comment:
this is on Account, not Contact, so I think the description and help text is wrong.